avm/res/app/job/main.bicep

```diff
@@ -0,0 +1,238 @@
+metadata name = 'Container App Jobs'
+metadata description = 'This module deploys a Container App Job.'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Required. Name of the Container App.')
+param name string
+
+@description('Optional. Location for all Resources.')
+param location string = resourceGroup().location
+
+@description('Required. Resource ID of environment.')
+param environmentId string
```

Comment:
@AlexanderSehr don't recall, did we eventually opt for having `resourceId` suffix instead of `Id`?

Reply:
Apparently so https://azure.github.io/Azure-Verified-Modules/specs/shared/#id-snfr22---category-inputs---parametersvariables-for-resource-ids

Reply:
We did, yes. Good call

Reply:
Fixed